src/main/include/hardware/ActuatorInterface.h

```diff
@@ -0,0 +1,29 @@
+
+#pragma once
+
+#include <memory>
+#include <rev/CANSparkMax.h>
+
+// Brushless Motor
+#define BRUSHLESS rev::CANSparkMax::MotorType::kBrushless
+
+// Motor IDs
+#define LEFT_MOTOR_1 1
```

Comment:
I do like the idea of having a centralized location for CAN/RIO port numbers. I suppose that's why we have this actuator interface?

Reply:
Yeah the idea I want to try is centralizing the hardware / port numbers and accessing them from the subsystems rather than initializing everything all over the place